### DISCUSSION TITLE
Unable to run ScriptZDebanding script

### DISCUSSION THREAD

**Discussion by M0J0-J0-J0** [sn4k3/UVtools#456] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/103470332/162857471-ec04ab75-6646-4d7f-8bd3-554238280150.png)

Getting this error (CS1061) when running this script. Using the Photon M3.

**Comment by sn4k3** [comment, marked as answer]
That's because you have an old script. You need to update/redownload.
Also that script is not longer needed. UVtools have that inbuilt under Suggestions tab: "Wait time before cure" which you can configure. So use it instead of the script

**Reply by M0J0-J0-J0** [reply]
Thank you so much for the reply!
